Sources/CombineSchedulers/TestScheduler.swift

```diff
@@ -262,20 +272,42 @@
       // NB: `DispatchTime(uptimeNanoseconds: 0) == .now())`. Use `1` for consistency.
       .init(now: .init(.init(uptimeNanoseconds: 1)))
     }
+    
+    /// A test scheduler compatible with type erased UI schedulers that simulates the inaccuracy of
+    /// real schedulers.
+    public static func inaccurate(
+      by minimumTolerance: SchedulerTimeType.Stride
```

Comment:
Can we make this an overload of `test`?

```suggestion
    @_disfavoredOverload
    public static func test(
      minimumTolerance: SchedulerTimeType.Stride
```

Reply:
Yes, that makes sense.